File: API/Models/MS_Shift.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace API.Models;

[PrimaryKey("Manuf", "Shift")]
[Table("MS_Shift")]
public partial class MS_Shift
{
    [Key]
    [StringLength(1)]
    public string Manuf { get; set; }

    [Key]
    [StringLength(1)]
    public string Shift { get; set; }

    [Required]
    [StringLength(50)]
    public string ShiftName { get; set; }
}

File: SPA/src/app/_core/models/maintain/msshift.ts
export interface MS_Shift {
  manuf: string;
  shift: string;
  shiftName: string
}
export interface ShiftDataMaintainParam {
  shift: string;
  shiftName: string;
}

export interface ShiftDataMaintainUploadParam {
  shift: string;
  file: File;
}
